### PR TITLE
Client: first stable release v0.1.0 🎉

### PR DESCRIPTION
**Pull request by holgerd77**
Ok, I guess that's it.

I wrote some more digestable release notes, updated the version number and the README and also did some last consistency checks (sync tests on all major networks e.g.).

Rebuilding the docs is not so trivial - I tried but failed - but I guess this is not so critical atm and we should skip for now.

Ready for review. 😄

Planning the release for tomorrow morning (European time).

Note: CHANGELOG best to read as a rendered [preview file](https://github.com/ethereumjs/ethereumjs-monorepo/blob/15eb2d4ab3866d9cee34d017f24dacd9bcf51f8a/packages/client/CHANGELOG.md).

### CODE REVIEW

File: packages/client/CHANGELOG.md [hunk 1]
```diff
@@ -6,50 +6,50 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
-- Save node key for consistent enode id, PR [#1067](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1067)
-- Refactor VM execution, PR [#1068](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1068)
-- Restart rlpx server on peerpool re-bootstrap, PR [#1113](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1113)
-- Split CLI transports options for bootnodes and multiaddrs, PR [#1145](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1145)
-- Simple message queue to handle unhandled ETH messages during handshake, PR [#1237](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1237)
-- bootstrap() extraction, PR [#1222](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1222)
-- Await DPT init, PR [#1233](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1233)
-- Make fetcher more typesafe, PR [#1023](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1023)
+After over a year of continued development the EthereumJS team is proud to announce that
+our client is finally as stable, feature-complete (or - let's call it: feature-consistent)
```
Comment (gabrocheleau) [inline packages/client/CHANGELOG.md:14]: It seems to me like there's a word/part of sentence missing in "is finally as stable [...]" (as stable as what?)

File: packages/client/CHANGELOG.md [hunk 2]
```diff
@@ -6,50 +6,50 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+and useful in many regards that we have decided to do a first official release! 🌼
 
-**Dependencies, CI and Docs**
+There have been done such an extensive amount of work pieces during that time that it
```
Comment (gabrocheleau) [inline packages/client/CHANGELOG.md:17]:
```suggestion
There has been done such an extensive amount of work pieces during that time that it 
```

File: packages/client/CHANGELOG.md [hunk 3]
```diff
@@ -6,50 +6,50 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
-- Make fetcher more typesafe, PR [#1023](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1023)
+After over a year of continued development the EthereumJS team is proud to announce that
+our client is finally as stable, feature-complete (or - let's call it: feature-consistent)
+and useful in many regards that we have decided to do a first official release! 🌼
 
-**Dependencies, CI and Docs**
+There have been done such an extensive amount of work pieces during that time that it 
+would be neither useful nor enlightening to list all associated PRs.
+
+So here are just a few milestones, where the ones till November 2020 still happened in
```
Comment (gabrocheleau) [inline packages/client/CHANGELOG.md:20]:
```suggestion
So here are just a few milestones, where the ones before November 2020 happened in
```

File: packages/client/CHANGELOG.md [hunk 4]
```diff
@@ -6,50 +6,50 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Upgraded `devp2p` `ETH` protocol up to version `ETH/66`, PR [#1331](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1331)
+- `LES` protocol v3 and v5 support, PR [#1324](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1324)
+- Support for `devp2p` snappy compression (RLPx v5), PR [#1399](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1399)
+- A simple miner for PoA/clique, PR [#1444](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1444)
+- Finalized on `devp2p` message handling support (e.g. by adding `NEW_BLOCK` rebroadcast capabilities) to serve as a good network citizen, PR [#1458](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1458)
+- Added developer capabilities, e.g. by adding a `--dev` flag to auto-start and prefund a miner, PR [#1492](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1492)
+
+And here we are. 🙂
+
+We joined various ephemeral testnets along the way - like `Yolov3` (still remember? 😃) or `Calaveras` - and helped find latest consensus bugs before they reach `mainnet` in preparation for the latest forks like `london` in conjunction with the other clients.
```
Comment (gabrocheleau) [inline packages/client/CHANGELOG.md:42]:
```suggestion
We joined various ephemeral testnets along the way - like `Yolov3` (still remember? 😃) or `Calaveras` - and helped find consensus bugs before they reach `mainnet` in preparation for the latest forks like `london` in conjunction with the other clients.
```

Seems more fluent to me without the duplicate 'latest'

File: packages/client/CHANGELOG.md [hunk 5]
```diff
@@ -6,50 +6,50 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+
+Now we are ready for the next step and hand this out to you - the community! 🥳
+
+We are eager to see what you will do with this client and how you will use this software, which for the first time in Ethereum's history brings the full Ethereum core protocol stack - complemented by the excellent [Lodestar](https://github.com/ChainSafe/lodestar) client ChainSafe is developing on the Eth2 side - into the hand of Node.js developers.
```
Comment (gabrocheleau) [inline packages/client/CHANGELOG.md:46]:
```suggestion
We are eager to see what you will do with this client and how you will use this software, which for the first time in Ethereum's history brings the full Ethereum core protocol stack - complemented by the excellent [Lodestar](https://github.com/ChainSafe/lodestar) client ChainSafe is developing on the Eth2 side - into the hands of Node.js developers.
```

File: packages/client/CHANGELOG.md [hunk 6]
```diff
@@ -6,50 +6,50 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+
+We have also updated our [README](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/client) on the client directly before the release which you should take as a comprise reference on the currently implemented client capabilities, some usages instructions complemented by various examples and developer notes.
```
Comment (gabrocheleau) [inline packages/client/CHANGELOG.md:48]: 'as a **comprise** reference'. Might that be a typo for 'concise'?

**Review by gabrocheleau — DISMISSED** [review]
Could of small text suggestions :)

File: packages/client/CHANGELOG.md (hunk 1 above)
Comment (ryanio) [reply packages/client/CHANGELOG.md:14]:
```suggestion
our client is finally stable, feature-complete (or - let's call it: feature-consistent)
```

File: packages/client/CHANGELOG.md (hunk 2 above)
Comment (ryanio) [reply packages/client/CHANGELOG.md:17]:
```suggestion
There has been such an extensive amount of work done that it
```

File: packages/client/CHANGELOG.md (hunk 4 above)
Comment (ryanio) [reply packages/client/CHANGELOG.md:42]:
```suggestion
We joined various ephemeral testnets along the way - like `Yolov3` (still remember? 😃) and `Calaveras` - and helped find consensus bugs before they reached `mainnet` in preparation for the latest forks like `london` in conjunction with the other clients.
```

File: packages/client/CHANGELOG.md (hunk 6 above)
Comment (ryanio) [reply packages/client/CHANGELOG.md:48]: not sure if meant "complete" or "concise"

```suggestion
We have also updated our [README](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/client) on the client directly before the release which you should take as a concise reference on the currently implemented client capabilities, some usages instructions complemented by various examples, and developer notes.
```

File: packages/client/README.md [hunk 7]
```diff
@@ -32,10 +30,10 @@ We invite you to explore and would be delighted if you give us feedback on your
 ## INSTALL
 
 ```shell
-npm install -g @ethereumjs/client // Release during Fall 2021
+npm install -g @ethereumjs/client
 ```
 
-As long as there is no up-to-date client release on npm and for development purposes the client can be used like this:
+For development purposes the client can be also used like this:
```
Comment (ryanio) [inline packages/client/README.md:36]:
```suggestion
For development purposes the client can also be set up like this:
```

**Comment by ryanio** [comment]
thanks @gabrocheleau for the review!

@holgerd77 please take a look at https://github.com/ethereumjs/ethereumjs-monorepo/pull/1499/commits/b7709feee3d48e27a9823aca6baa3a431813060c and confirm all our changes look OK to you - and feel free to update further.

**Review by ryanio — DISMISSED** [review]

**Review by ryanio — APPROVED** [review]

File: packages/client/CHANGELOG.md [hunk 8]
```diff
@@ -6,50 +6,50 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [UNRELEASED]
+## 0.1.0 - 2021-09-29
 
-**New Features**
+Heydiho, glad you are here! 😀
```
Comment (jochem-brouwer) [inline packages/client/CHANGELOG.md:11]: :tada: :smile:

**Review by jochem-brouwer — APPROVED** [review]
Awesome!

**Comment by holgerd77** [comment]
Just published the following releases on npm:

- `@ethereumjs/client` [v0.1.0](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fclient%400.1.0)

**Pull request merged**